tools/project_manager/manage_image_sources.php

```diff
@@ -384,7 +384,9 @@ function _show_edit_permissions_row()
             ? (empty($_REQUEST[$field]) ? '2' : $_REQUEST[$field])
             : $this->$field;
 
-        $editing .= "<div class='perms_wrapper'>" . _("Visibility on Info Page") . "</div> <select name='$field'>";
+        $editing .= _("Visibility on Info Page: ");
+        $editing .= "<select name= '$field'>";
```

Comment:
space after = could be removed. Also, we could attribute encode field, but seems safe currently.

Reply:
Thanks for catching that extra space.

I've added `attr_safe`, but please bear in mind that I will need to be told where and when to add it until I get a better handle on when and where it's appropriate. And, given that I've seen different patterns within the code, probably how, as well. 😁

Reply:
I think the idea is that if you don't control the content, you need to be prepared it may have invalid characters in regards to being used as an attribute (characters such as ' or " which could allow XSS attacks. In this case it appears we control it, but there is no harm in encoding something that won't change after encoding so it is best (in my opinion) to just always encode.